Part 1 commit, 10 nouns and 3 verbs per noun

gavinfinn opened the pull request:
Slack username:

gavinfinn committed:
Part 1 commit, 10 nouns and 3 verbs per noun

matkatmusic commented on main.cpp:
```diff
@@ -51,107 +51,108 @@ arm.flex();                             // 3)
-
-//  action 3:
-
+//  Noun: baker
+//  action 1: the baker kneads the dough
+baker.knead();
+//  action 2: the baker bakes a cake
+baker.bakeCake();
+//  action 3: the baker decorates the cake
+baker.decorateCake();
```
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

matkatmusic commented on main.cpp:
```diff
@@ -51,107 +51,108 @@ arm.flex();                             // 3)
+//  Noun: pianist
+//  action 1: the pianist plays a C Maj chord   
+pianist.playCMaj();
+//  action 2: the pianist plays a glissando
+pianist.gliss();
```
`gliss` is what the piano player plays.  
- [x] revise: `playGliss();`

matkatmusic commented on main.cpp:
```diff
@@ -51,107 +51,108 @@ arm.flex();                             // 3)
-
+//  Noun: policeman
+//  action 1: the policeman patrols
+policeman.patrol
```
missing the `();`
- [x] revise

gavinfinn committed:
PFMCPP-Project 1 Review Updates

Modified Noun 8 to a new object
Fixed missing parentheses on line  122
Revised function name on line 83
